adv_project/settings.py

```diff
@@ -26,7 +28,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = config('DEBUG', cast=bool)
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = os.environ.get("ALLOWED_HOSTS").split(",")
```

Comment:
This is interesting, you could also have casted to a csv() type to get similar functionality.